XenModel/Messages.resx

```diff
@@ -10556,6 +10571,15 @@ Please reconnect the host and try again</value>
   <data name="PROBLEM_HOSTPROBLEM_TITLE" xml:space="preserve">
     <value>Server '{0}'</value>
   </data>
+  <data name="PROBLEM_VSWITCH_CONTROLLER_DESCRIPTION" xml:space="preserve">
+    <value>{0}: Support for the vSwitch Controller is dropped as of {1}</value>
+  </data>
+  <data name="PROBLEM_VSWITCH_CONTROLLER_HELP" xml:space="preserve">

```

Comment:
This one I'll probably have to remove, it may not be possible to deconfigure it via XenCenter.